.strigo/config.yml

```diff
@@ -1,45 +1,88 @@
-## Class exercises - simple configuration for a single class
+## Class lab resources and exercises - simple configuration for a single class
 ---
 classes:
-  - exercises:
-      # `file` is relative to .strigo directory
+  - lab_resources:
+      - display_name: "AWS EC2 Instance"
+        server_image_id: "ami-0abcdef1234567890"
+        platform_type: "lab"
+        image_platform: "linux"
+        cloud_provider: "aws"
+        view_interface: "Desktop"
+        instance_type: "t2.micro"
+      - display_name: "Web Resource"
+        web_resource_url: "https://example.com"
+        platform_type: "web"
+    exercises:
       - file: "../hello-world/hello.md"
         title: "Hello!"
       - file: "../hello-world/world.md"
         title: "World!"
 
-## Class exercises - configuration for multiple classes
+## Class lab resources - configuration for multiple classes
 # ---
 # classes:
-#   # `class_id` may be omitted if there is only one class
-#   # in this configuration file
 #   - class_id: "<your_class_id>"
+#     lab_resources:
+#       - display_name: "AWS EC2 Instance"
+#         server_image_id: "ami-0abcdef1234567890"
```

Comment:
server_image_id is describing a strigo document from the server-images collection and not an AMI id.
I think this should be omitted.

for specifying the image Id, we use the ami_region_mapping.
in the public API we also accommodate for a singular image_id, which we transform into the amiRegionMapping field.